samples/charts/data-chart/stacked-100-bar-chart/App.razor

```diff
@@ -0,0 +1 @@
+<Infragistics.Samples.Pages.Stacked100BarChart />
```

Comment:
replace this line with content of Stacked100BarChart.razor file

Reply:
done